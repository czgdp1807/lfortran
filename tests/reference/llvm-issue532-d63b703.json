{
    "basename": "llvm-issue532-d63b703",
    "cmd": "lfortran --no-color --show-llvm {infile} -o {outfile}",
    "infile": "tests/issue532.f90",
    "infile_hash": "75f4785bea0a877f7a902640ffcc5d276bd955349ff3df1639d6e6b7",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "llvm-issue532-d63b703.stdout",
    "stdout_hash": "35ba4e068f494bad4d9a08e68a1aa79694b9e4bdb37670fb6b2c71a5",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}