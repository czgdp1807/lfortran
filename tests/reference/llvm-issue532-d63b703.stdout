; ModuleID = 'LFortran'
source_filename = "LFortran"

declare void @__module_lfortran_intrinsic_builtin_execute_command_line(i8**, i1*, i32*, i32*, i8**)

declare void @__module_lfortran_intrinsic_builtin_get_command_argument(i32*, i8**, i32*, i32*)

declare void @__module_lfortran_intrinsic_builtin_get_environment_variable(i8**, i8**, i32*, i32*, i1*)

declare void @__module_lfortran_intrinsic_builtin_move_alloc(i32*, i32*)

define i32 @main() {
.entry:
  ret i32 0
}
